nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSSession.java

```diff
@@ -363,19 +363,41 @@ protected Document getChild(String parentId, String name) {
         return children;
     }
 
-    protected List<String> getChildrenIds(String parentId) {
+    protected List<String> getAllChildrenIds(String parentId) {
+        // We want all children, the filter flags are null
+        boolean excludeSpecialChildren = false;
+        boolean excludeRegularChildren = false;
+        return internalGetChildrenIds(parentId, excludeSpecialChildren, excludeRegularChildren);
+    }
+
+    protected List<String> getChildrenIds(String parentId, boolean excludeSpecialChildren,
+            boolean excludeRegularChildren) {
+        return internalGetChildrenIds(parentId, excludeSpecialChildren, excludeRegularChildren);
+    }
+
+    protected List<String> internalGetChildrenIds(String parentId, boolean excludeSpecialChildren,
```

Comment:
You can now rename this to `getChildrenIds` directly, the two methods are the same.